dbt_subprojects/dex/models/trades/dex_trades.sql

```diff
@@ -61,7 +61,7 @@ WITH curve AS (
     {{
         enrich_balancer_v3_dex_trades(
             base_trades = ref('dex_base_trades')
-            , filter = "(project = 'balancer' AND version = '3')"
+            , filter = "(project IN ('balancer', 'beets') AND version = '3')"
```

Comment:
i think we then also need to filter out in below `dexs` CTE to match balancer?
as this design pattern grows, maybe we can find a more efficient way to write this model 🤔